config/detekt/detekt.yml

```diff
@@ -776,6 +776,7 @@ style:
     allowedNames: '(_|ignored|expected|serialVersionUID)'
     ignoreAnnotated:
       - 'Preview'
+      - 'PreviewComposable'
```

Comment:
@BottleRocket-Colin Nice! I ended up making some changes in [my PR](https://github.com/BottleRocketStudios/Android-ArchitectureDemo/pull/88) to cleanup the codebase and I forgot about this one. I'll update my PR once this one is merged in to use the latest and greatest changes you've incorporated.